vault/changelog.release.2022-07-26.md

```diff
@@ -0,0 +1,74 @@
+---
+id: xhcrzg39oqahvb89jb2m1ur
+title: 0.105
+desc: ''
+updated: 1658762256849
+created: 1658758655969
+---
+
+Dendron 0.105 has sprouted  🌱
+July 26, 2022
+
+**New Display Options in Graph Panel:** You can now toggle the display of linked and hierarchal edges in graph panel.
+
+- More information: [[Toggle Edges|dendron://dendron.dendron-site/dendron.topic.sidebar.graph-panel#toggle-edges]]
+
+<div style="position: relative; padding-bottom: 53.28125%; height: 0;"><iframe src="https://www.loom.com/embed/88c169c981244c289dde63eac2419bd1" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
+
+## Highlights
+
+- enhance(views): option to filter hierarchical edges from graph panel
+
+## Everything Else
+
+- enhance(publish): use note's `desc`field for SEO description 
+- enhance(retrieve): support end block anchor
+- fix(workspace): information modal to uninstall 'dendron markdown links' extension
+- fix(publish): hide sidebar when clicking a non-submenu note on mobile
+- fix(publish): current menu item on the sidebar wont collapse for a published site
+- fix(views): properly focus tree view to active note when it is first shown 
+- fix(workspace): fix typos in getting started tutorial
+
+
+## Community
+
+### General Updates
```

Comment:
add another reminder for greenhouse talk this week with details from https://airtable.com/appv7r4uhFzqTvj5m/tbl9oZ3f3910235Rp/viwoEco7HvQxBd9AN/recM0LukZ8JAaQ8v7?blocks=hide